app/screens/channel/header/header.tsx

```diff
@@ -87,10 +89,13 @@ const ChannelHeader = ({
     const theme = useTheme();
     const styles = getStyleSheet(theme);
     const defaultHeight = useDefaultHeaderHeight();
+    const serverUrl = useServerUrl();
+
+    const callsConfig = getCallsConfig(serverUrl);
 
     // NOTE: callsEnabledInChannel will be true/false (not undefined) based on explicit state + the DefaultEnabled system setting
     //   which ultimately comes from channel/index.tsx, and observeIsCallsEnabledInChannel
-    let callsAvailable = callsEnabledInChannel;
+    let callsAvailable = callsConfig.pluginEnabled && callsEnabledInChannel;
```

Comment:
I'm undecided over whether or not to move the `callsConfig.pluginEnabled` check into the `observeIsCallsEnabledInChannel`, so that `callsEnabledInChannel` will be false if the plugin is not enabled, which makes sense... But you could also argue that it combines two different states: whether or not calls are active and whether calls have been explicitly enabled/disabled/default. So I'm fine either way.